src/fetch/payouts.py

```diff
@@ -418,6 +425,19 @@ def construct_payout_dataframe(
         merged_df["eth_slippage_wei"].fillna(0) + merged_df["network_fee_eth"]
     )
 
+    # 5. Compute buffer accounting target
+    merged_df["buffer_accounting_target"] = np.where(
+        merged_df["pool_address"] != COW_BONDING_POOL.address,
+        merged_df["solver"],
+        merged_df["reward_target"],
+    )
+
+    # 6. Add reward token address
+    merged_df["reward_token_address"] = COW_TOKEN_ADDRESS.address
+
+    # 7. Missing service fee is treated as new solver
+    merged_df["service_fee"] = merged_df["service_fee"].fillna(Fraction(0, 1))  # type: ignore
```

Comment:
Is there a way to have a log here that prints which solver addresses didn't have a service fee associated with them?